fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useEffect, useState } from 'react';
+import PageTitle from './PageTitle';
+import TodoList from './TodoList';
+import TodoForm from './TodoForm';
+
+export default function Todos() {
+  const [todos, setTodos] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+
+  /* Implement useEffect to fetch all todos. Hints are at the bottom of the file. */
+  /* useEffect:
+   *  Use fetch to send a GET request to `/api/todos`.
+   *  Once the response JSON is received and parsed,
+   *  update state with the received todos.
+   *  Handle errors correctly and set `error` to any error or error status code.
+   *  Set `isLoading` to false when all network activity is finished.
+   */
+  useEffect(() => {
+    async function readJSONTodos() {
+      try {
+        const response = await fetch('/api/todos');
+        if (!response.ok) {
+          throw new Error('Network response was not OK', response.status);
+        }
+        const jsonData = await response.json();
+        setTodos(jsonData);
+      } catch (error) {
+        setError(error);
```

Comment:
Set isLoading to false in this branch, too, or the user will never see the error message.